Wikipedia/Code/StorageAndSyncingSettingsViewController.swift

```diff
@@ -0,0 +1,213 @@
+struct SettingsSection {
+    let headerTitle: String?
+    let footerText: String?
+    let items: [SettingsItem]
+}
+
+struct SettingsItem {
+    let disclosureType: WMFSettingsMenuItemDisclosureType?
+    let itemType: StorageAndSyncingSettingsItemType
+    let title: String?
+    let iconName: String?
+    let iconColor: UIColor?
+    let iconBackgroundColor: UIColor?
+    let isSwitchOn: Bool?
```

Comment:
`isSwitchOn` would be simpler if not optional - just default it to false and use ` ?? false`  at the end of the `let showSavedReadingList = ` line.